playground/milvus/MilvusPlayground.AppHost/Program.cs

```diff
@@ -0,0 +1,12 @@
+var builder = DistributedApplication.CreateBuilder(args);
+
+var token = builder.AddParameter("milvusauth", true);
```

Comment:
Is this necessary? Can we just generate an auth token instead?

Reply:
No. there is only one user in a default Milvus container (root:Milvus). This cannot be configured presently via environment variables or anything and can only be configured after the fact (or additional users after the fact). See: https://github.com/milvus-io/milvus/issues/33058 and https://github.com/milvus-io/milvus/discussions/33032#discussioncomment-9427837

Reply:
How's this code going to work then?

https://github.com/dotnet/aspire/blob/0a3cbecc3a8754a8b45e3ab6d5548854e6779d05/src/Aspire.Hosting.Milvus/MilvusBuilderExtensions.cs#L35-L37

Reply:
Me thinking aloud why I kept this in...

It *is* possible to reset the root token.  But right now it has to be done after the fact. So in theory someone could do a data volume, have a script (or some other client operation) that changes the root login (or adds another user), and then this would need to be changed (they would populate the parameter.  Correct here that the CreateDefaultPasswordParameter wouldn't really work in this instance.  I was also trying to avoid hard-coding the default root creds here.

To me the ideal would look like `apiKey?.Resource ?? DefaultMilvusApiKeyParameter` - per discussion in teams channels, there is no easy way to create a ParameterResource with a string value default.

Thoughts welcome here...

Reply:
@timheuer - thinking more about this - how is this going to work in a deployed/production environment? If I use `azd` to deploy an app that has a Milvus resource, will the ApiKey always be `root:Milvus` even in a deployed environment? That doesn't seem secure.